core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html

```diff
@@ -66,33 +60,36 @@
                             'com.dotcms.contenttype.model.field.ImmutableWysiwygField'
                     "
                     [currentFieldType]="currentFieldType"
-                    (convert)="convertWysiwygToBlock($event)"
-                ></dot-convert-wysiwyg-to-block>
+                    (convert)="convertWysiwygToBlock($event)"></dot-convert-wysiwyg-to-block>
             </div>
         </p-tabPanel>
-        <p-tabPanel
-            #panel
-            *ngIf="!!currentField?.id && isBlockEditorField"
-            [header]="'Settings'"
-            [disabled]="!currentField?.id"
-        >
+        @if(!!currentField?.id && isFieldWithSettings) {
+        <p-tabPanel #panel [header]="'Settings'" [disabled]="!currentField?.id">
+            @switch ( this.currentFieldType?.clazz) { @case
+            ('com.dotcms.contenttype.model.field.ImmutableStoryBlockField') {
             <dot-block-editor-settings
                 [field]="currentField"
                 [isVisible]="panel._selected"
                 (changeControls)="changesDialogActions($event)"
                 (save)="toggleDialog()"
+                (valid)="setDialogOkButtonState($event)"></dot-block-editor-settings>
+            } @case("com.dotcms.contenttype.model.field.ImmutableBinaryField") {
+            <dot-binary-settings
+                [field]="currentField"
+                [isVisible]="panel._selected"
+                (save)="toggleDialog()"
                 (valid)="setDialogOkButtonState($event)"
-            ></dot-block-editor-settings>
+                (changeControls)="changesDialogActions($event)"></dot-binary-settings>
+            }}
         </p-tabPanel>
+        }
```

Comment:
It's not for this PR. However, we may need to create a `wrapper` if more fields have `settings`.